### PR TITLE
Fixes easy money plug on CS case study

### PR DESCRIPTION
**Pull request by joaojusto**
Old: 

screen shot 2015-08-05 at 14 09 27

New:

screen shot 2015-08-05 at 14 09 14

### CODE REVIEW

File: source/stylesheets/components/_panel.scss [hunk 1]
```diff
@@ -253,17 +253,18 @@
 
 .Panel.Panel--easymoneyMapaMundi {
   position: relative;
+  z-index: 1;
```
Comment (azevedo-252) [inline source/stylesheets/components/_panel.scss]: What are these z-index doing?

File: source/stylesheets/components/_panel.scss [hunk 2]
```diff
@@ -253,17 +253,18 @@
...
 
-  height: 0;
```
Comment (azevedo-252) [inline source/stylesheets/components/_panel.scss]: Did you check if this change didn't affect negatively other places where this component is being used?

File: source/stylesheets/components/_panel.scss (hunk 1 above)
Comment (joaojusto) [inline source/stylesheets/components/_panel.scss]: Without this z-index, the map is getting on top of the button.

File: source/stylesheets/components/_panel.scss (hunk 2 above)
Comment (joaojusto) [inline source/stylesheets/components/_panel.scss]: I did check yes

**Comment by azevedo-252** [comment]
:+1:

**Pull request merged**